Framework/Core/src/ExternalFairMQDeviceProxy.cxx

```diff
@@ -450,6 +455,11 @@ DataProcessorSpec specifyFairMQDeviceOutputProxy(char const* name,
     callbacks.set(CallbackService::Id::Start, channelConfigurationChecker);
     auto lastDataProcessingHeader = std::make_shared<DataProcessingHeader>(0, 0);
 
+    for (auto const& inputSpec : inputSpecs) {
```

Comment:
even if we introduce a different way to introduce the forward routes, we have to check that for all inputs only forwards to the external channels are defined.